application/Profile/LocalProfile.py

```
@@ -88,33 +105,41 @@
-            print('Not valid SSH key found for deletion')
+            logger.exception('No valid SSH key found for deletion')
 
         self._profile['sessions'].pop(session_id)
         self.save_profile()
 
+        logger.info(f'Successfully deleted session {session_id}')
+
         return True, ''
 
-    def change_host(self, session_id, new_host):
+    def change_host(self, session_id, new_host) -> Tuple[bool, str]:
```

Ruff (0.8.2)

Missing type annotation for function argument `session_id`

(ANN001)